scripts/check-rate-apigw/README.md

```diff
@@ -0,0 +1,91 @@
+## Check Rate Limit Api-Gw
+
+this script test the rate limit of api-gw, the rate limit (rate + burst) must be edited outside this script:
+
+usage:
+
+`./checkrateapigw.sh`
+
+and than insert api-key.
+
+Test report:
+
+1 - TEST WITH DEFAULT PARAMITERS; RATE 100 AND BURST 200:
```

Comment:
@mbrachi separa il report in un altro file

Reply:
@fdammassa report separato, standardizzata la produzione del file e aggiunto il file sample_report.txt per tenere traccia della prova effettuata.